Test precision failure on Ubuntu autopkgtests with glibc 2.29

We are seeing the following failure only on ppc64el architecture when the tests for 1.73 are run against new glib 2.29 on Ubuntu autopkgtests.

http://autopkgtest.ubuntu.com/packages/p/python-biopython/disco/ppc64el 

FAIL: test_WellRecord (test_phenotype_fit.TestPhenoMicro)
Test basic functionalities of WellRecord objects
Traceback (most recent call last):
  File "/tmp/autopkgtest.6H8J2c/autopkgtest_tmp/Tests/test_phenotype_fit.py", line 55, in test_WellRecord
    self.assertAlmostEqual(w.plateau, 188.51404344898586, places=5)
AssertionError: 188.51403793245183 != 188.51404344898586 within 5 places

I wonder it it would be acceptable to fix this by reducing the required precision here, similar to what was done for #1038?

Thank you.

--- a/Tests/test_phenotype_fit.py
+++ b/Tests/test_phenotype_fit.py
@@ -56,7 +56,7 @@ class TestPhenoMicro(unittest.TestCase):
         self.assertAlmostEqual(w.area, 20879.5)
         self.assertEqual(w.model, "gompertz")
         self.assertAlmostEqual(w.lag, 6.0425868725090357, places=5)
-        self.assertAlmostEqual(w.plateau, 188.51404344898586, places=5)
+        self.assertAlmostEqual(w.plateau, 188.51404344898586, places=4)
         self.assertAlmostEqual(w.slope, 48.190618284831132, places=4)
         self.assertAlmostEqual(w.v, 0.10000000000000001, places=5)
         self.assertAlmostEqual(w.y0, 45.879770069807989, places=5)

